src/browser-polyfill.js

```diff
@@ -6,7 +6,11 @@
  * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
 "use strict";
 
-if (typeof browser === "undefined" || Object.getPrototypeOf(browser) !== Object.prototype) {
+if (typeof globalThis != "object" || typeof chrome != "object" || !chrome || !chrome.runtime || !chrome.runtime.id) {
```

Comment:
Why is this failing now? The tests seem to be running on Chrome 90. `globalThis` was added 3 years ago in v71

Reply:
@fregante I just looked to the failures in the last CI job and in the last one failed on the `npm install` step because most of the unit tests did fail.

I suspected that it was because of the nodejs version that is currently being used in the CI jobs, and that seems confirmed by the fact that `globalThis` has been introduced in nodejs 12 ([compatibility table on MDN](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/globalThis#browser_compatibility)) and the fact that in the current CI config the nodejs version is definitely set to v10.

I just opened #353 to bump the nodejs version to v12 in the CI config, as soon as I have done fighting with the chromedriver related failure I'm going to merge it. 

After that, you may rebase this PR to trigger the CI job on nodejs v12 and confirm if that was the case.

Reply:
@fregante I merged #354, and so now the CI jobs from any branch based on that CI config should be using nodejs v12, and so you can give it a try.

Reply:
That worked!